### PR TITLE
add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by heleeen**
I added CONTRIBUTING.md and Makefile to standardize test command.

Please review it 🙏

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+TF_ACC=1
```
Comment (hayajo) [inline Makefile:1]:
```suggestion
export TF_ACC=1
```

Comment (heleeen) [reply Makefile]: thanks for your suggestion!

**Review by hayajo — APPROVED** [review]
LGTM!

**Comment by heleeen** [comment]
Thanks for your review!

**Pull request merged**